ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml

```diff
@@ -252,6 +252,10 @@ tests:
   postsubmit: true
   steps:
     workflow: stackrox-automation-flavors-aks-e2e
```

Comment:
I think `timeout: 5h0m0s` needs to be in this step as well. I'm pretty sure that's why it's removed in the postsubmits.yaml. Otherwise, lgtm